Dockerfile

```diff
@@ -1,12 +1,28 @@
-FROM node:15.14.0-alpine3.13
+ARG APP_DIR="/usr/app"
+ARG ENDPOINT_PORT="4000"
 
+# Stage 1
+FROM node:15.14.0-alpine3.13 AS installer
+ARG APP_DIR
+ENV APP_DIR=$APP_DIR
+WORKDIR ${APP_DIR}
+
+COPY package*.json ./
 RUN apk add --update make \
-    && npm install --global npm@7.8.0
+    && npm install
```

Comment:
why the removal of `--global npm@7.8.0`?

Reply:
npm updated to 7.9.0 and the complaining came back.

Because of that I realized we'd be constantly chasing npm versions and still wouldn't be able to silence those warnings for more than a few days at a time. Might as well just default to the npm that comes with the base image.